### ISSUE TITLE
Can longclaw create a b2b e-commerce platform?

### ISSUE THREAD

**Issue by nokemono** [JamesRamm/longclaw#442]
Can longclaw create a b2b e-commerce platform?
In other words, I want to allow users to post their products.

**Comment by chrisbarry** [comment]
Yes, no reason it couldn't be used to do this, but I would possibly just build it from scratch. You could look at wagtail multi site so it can segregate different admins from different users, but the wagtail admin might be too flexible in a lot of regards for a shopify like product...

**Comment by JamesRamm** [comment]
It could be, but it would need an amount of work 'on top' of it. 
You would need to implement some sort of tenancy probably so that each user can only access their own data in the wagtail admin. I dont think there is this kind of separation today (I dont know if wagtail has anything built-in in that regards)
